samples/core/console-apps/Fibonacci/README.md

```diff
@@ -1,7 +1,7 @@
 Fibonacci Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```

Comment:
Wouldn't these be more resilient if we targeted the markdown using a relative link?
e g: ../../tutorials/using-with-xplat-cli.md